packages/app/run_android.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+PORT=1420
+DX_OUTPUT_DIR="../../target/dx/main-ui/release/web/public"
```

Shellcheck (0.10.0)

[warning] DX_OUTPUT_DIR appears unused. Verify use (or export if used externally).

(SC2034)